src/AsyncTestCase.php

```diff
@@ -146,4 +149,14 @@ abstract class AsyncTestCase extends PHPUnitTestCase
 
         return $mock;
     }
+
+    protected function setUpAsync(): Promise
```

Comment:
Not sure whether we should use this name or something like beforeEach.

Reply:
I'm against, because those names (without Asnyc suffix) are used by PHPUnit and people will immediately understand what those are used for.